matplotlibcpp.h

```diff
@@ -500,6 +500,101 @@ void plot_surface(const std::vector<::std::vector<Numeric>> &x,
   if (res) Py_DECREF(res);
 }
 
+template <typename Numeric>
+void plot_scatter(const std::vector<::std::vector<Numeric>> &x,
+                  const std::vector<::std::vector<Numeric>> &y,
+                  const std::vector<::std::vector<Numeric>> &z)
+{
+    
+    //const std::map<std::string; 
```

Comment:
Is this PR ready?  There seems to be a lot of dead code.

Reply:
sorry, forgot to delete irrelevant code taken from another example.  I used this plot_scatter function in my code and it's working correctly...